### DISCUSSION TITLE
World missing from list

### DISCUSSION THREAD

**Discussion by MaltySalty** [Pryaxis/TShock#1917] [General]
I have installed the latest version of terraria and I'm using the latest version of the Tshock server but I now can't see all of my worlds (especially the world that I want to use). Is there any way to fix this or is this an issue that hasn't been fixed yet?

**Comment by kubedzero** [comment]
Can you provide any further information, such as your operating system, the folder on your computer where your world files are stored, how you're running TShock, what TShock version you're using? 

I haven't seen this issue before but my guess is that you'd be able to fix it by running TShock with a direct path to the world you want. You could make a `launch.bat` file, open it in a text editor, and add `C:\Users\Win\Downloads\tShock\TerrariaServer.exe -port 7778 -world "C:\Users\Win\My Documents\My Games\Terraria\Worlds\myWorld.wld" ` and it should launch as expected with the world you've chosen.

**Reply by MaltySalty** [reply]
that fixed it. For some reason it didnt choose the correct path. Thanks!
